de.metas.adempiere.adempiere/base/src/main/java/de/metas/security/UserAuthTokenRepository.java

```diff
@@ -96,26 +96,22 @@ private static String generateAuthTokenString()
 		return UUID.randomUUID().toString().replace("-", "");
 	}
 
-	public UserAuthToken retrieveByUserId( @NonNull final UserId userId,  @NonNull final RoleId roleId)
+	public UserAuthToken retrieveByUserIdOrNull( @NonNull final UserId userId,  @NonNull final RoleId roleId)
 	{
 		final List<I_AD_User_AuthToken> userAuthTokens = Services.get(IQueryBL.class)
 				.createQueryBuilder(I_AD_User_AuthToken.class)
 				.addOnlyActiveRecordsFilter()
 				.addEqualsFilter(I_AD_User_AuthToken.COLUMN_AD_User_ID, userId)
-				.addEqualsFilter(I_AD_User_AuthToken.COLUMN_AD_Role_ID, roleId)
-				.setLimit(2)
+				.addInArrayFilter(I_AD_User_AuthToken.COLUMN_AD_Role_ID, roleId, RoleId.ADMIN)
```

Comment:
```suggestion
				.addEqualsFilter(I_AD_User_AuthToken.COLUMN_AD_Role_ID, roleId)
```

I think it's dangerous to fall back to one of the most powerfull roles we have in metasfresh.
If we can't find a better solution then imho we need so create a dedicated role for the JsonReports user and specify that, same way we do specify the jasper-json-user itself